src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -26,12 +26,15 @@ object TezosNodeOperator {
     */
   final case class OperationResult(results: AppliedOperation, operationGroupID: String)
 
+  val isGenesis = (data: BlockData) => data.header.level == 0
+
 }
 
 /**
   * Operations run against Tezos nodes, mainly used for collecting chain data for later entry into a database.
   */
 class TezosNodeOperator(val node: TezosRPCInterface, batchConf: BatchFetchConfiguration)(implicit executionContext: ExecutionContext) extends LazyLogging {
+  import TezosNodeOperator.isGenesis
```

Comment:
Is the reason you define isGenesis out of the scope of TezosNodeOperator because it's not intrinsic to interacting with the Tezos Node?

Reply:
Just because it's a generic function that can be reused, and I want to define it only in a single place